client/state/account-recovery/reset/reducer.js

```diff
@@ -83,11 +86,30 @@ const validate = combineReducers( {
 	} ),
 } );
 
+const resetPassword = combineReducers( {
+	isRequesting: createReducer( false, {
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST ]: stubTrue,
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS ]: stubFalse,
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR ]: stubFalse,
+	} ),
+	succeeded: createReducer( false, {
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST ]: stubFalse,
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS ]: stubTrue,
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR ]: stubFalse,
+	} ),
+	error: createReducer( null, {
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST ]: () => null,
+		[ ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS ]: () => null,
```

Comment:
Similar to [my reply above](https://github.com/Automattic/wp-calypso/pull/13364#discussion_r114713565). The test for `resetPassword.error` begins at [line 319](https://github.com/Automattic/wp-calypso/pull/13364/files/c6b8e731be3f44da007f7ad9c54116ba395a5c8c#diff-9946f7256187790c55dbd69fd2cef3fcR319)